pkg/kubelet/status/status_manager.go

```diff
@@ -261,28 +266,32 @@ func (m *manager) SetContainerStartup(podUID types.UID, containerID kubecontaine
 
 	pod, ok := m.podManager.GetPodByUID(podUID)
 	if !ok {
-		klog.V(4).Infof("Pod %q has been deleted, no need to update startup", string(podUID))
+		klog.V(4).InfoS("Pod has been deleted, no need to update startup", "pod", string(podUID))
 		return
 	}
 
 	oldStatus, found := m.podStatuses[pod.UID]
 	if !found {
-		klog.Warningf("Container startup changed before pod has synced: %q - %q",
-			format.Pod(pod), containerID.String())
+		klog.InfoS("Container startup changed before pod has synced",
+			"pod", klog.KObj(pod),
+			"containerID", containerID.String())
 		return
 	}
 
 	// Find the container to update.
 	containerStatus, _, ok := findContainerStatus(&oldStatus.status, containerID.String())
 	if !ok {
-		klog.Warningf("Container startup changed for unknown container: %q - %q",
-			format.Pod(pod), containerID.String())
+		klog.InfoS("Container startup changed for unknown container",
+			"pod", klog.KObj(pod),
+			"containerID", containerID.String())
 		return
 	}
 
 	if containerStatus.Started != nil && *containerStatus.Started == started {
-		klog.V(4).Infof("Container startup unchanged (%v): %q - %q", started,
-			format.Pod(pod), containerID.String())
+		klog.V(4).InfoS("Container startup unchanged",
+			"started", started,
```

Comment:
I don't think there is a need to keep "started" variable.  This value cannot be joined with other logs as it has context only to this log. Also this variable is always expected to have same value `containerStatus.Started`, if you look into if statement it checks for this value.
```suggestion
```